src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidContentProviderBroker.cs

```diff
@@ -345,9 +344,9 @@ private string GetContentProviderURIForOperation(string operation)
             return BrokerConstants.MsAuthenticatorContentProviderUri + "/" + operation;
         }
 
-        private string SearializeBundleToJSON(Bundle bundle)
+        private string SerializeBundle(Bundle bundle)
         {
-            return Base64UrlHelpers.Encode(Marshall(bundle));
+            return Convert.ToBase64String(Marshall(bundle));
```

Comment:
This is the fix. Everything else is renamings.

Reply:
Very cool. How did you figure it out?

Reply:
Basically looked at all the code where we encode the values that we pass in the content resolver query. Then repro'd by faking the broker request with the client Id and redirect URI of the app that was breaking.